Makefile

```diff
@@ -270,14 +272,28 @@ kind-down: $(KIND)
 	rm -f $(REPO_ROOT)/example/provider-local/base/kubeconfig
 	rm -rf dev/local-backupbuckets
 
+
+kind-seed-up: $(KIND)
+	@if [ "$(KIND_ENV)" = "local" ]; then ip addr add 127.0.0.2 dev lo0; fi
```

Comment:
Hmm, should we clean up the added IP address somewhere?
Also, the loopback interface might be named differently than `lo0`.

Reply:
Moved it out of the makefile and added as an additional step in the documentation. I tried a way to get the name of the local interface from `ip link show` but the output seems to be in a bit different formats depending on the OS. I think it isn't something that will be executed too often so it could be ok only as an additional step.